py-polars/polars/functions/eager.py

```diff
@@ -31,14 +31,14 @@ def concat(
     parallel: bool = True,
 ) -> PolarsType:
     """
-    Combine multiple DataFrames, LazyFrames, or Series into a single object.
+    Combine multiple DataFrames, LazyFrames, Series, or Expr into a single object.
 
     Parameters
     ----------
     items
-        DataFrames, LazyFrames, or Series to concatenate.
+        DataFrames, LazyFrames, Series, or Expr to concatenate.
     how : {'vertical', 'vertical_relaxed', 'diagonal', 'diagonal_relaxed', 'horizontal', 'align'}
-        Series only support the `vertical` strategy.
+        This is not used when items are Expr. Series only support the `vertical` strategy.
```

Comment:
I think `vertical` should be checked for Expr as well.

Reply:
```suggestion
        Series and Expr only support the `vertical` strategy.
```